### DISCUSSION TITLE
REST API calls to Origin dont always work in multi-server setup

### DISCUSSION THREAD

**Discussion by bartvp1** [OvenMediaLabs/OvenMediaEngine#1753] [Ideas]
**Describe the bug**
When trying to call any endpoint of the Origin REST API while running multiple Origin instances behind a load balancer, the call might end up in the wrong instance, causing the request to fail.

**To Reproduce**
1. Create a setup with multiple Origin servers
2. Start a livestream
3. Try to start recoding the stream by calling the REST endpoint on the Origin server which is not processing the livestream.
4. The response indicates the livestream doesn't exist

**Expected behavior**
I would expect OME to start recording the livestream, regardless of which Origin instance receives the request.

**Logs**
_Not applicable_

**Server**
 - OS: Ubuntu
 - OvenMediaEngine Version: `v0.17.1`

**Additional context**
OME should be able to figure out where the REST call should be targeted by using the IP/hostname in the `OriginMapStore`

**Comment by getroot** [comment]
Great idea. But I don't think this is something that the Origin server should do. I think it's better to do it on the WAS server that calls the API. I think OvenMediaEngine is better focused on streaming.
